src/components/animated-robot-hands.js

```diff
@@ -0,0 +1,97 @@
+// Global THREE, AFRAME
+const POSES = {
+  open: "allOpen",
+  thumbDown: "thumbDown",
+  indexDown: "indexDown",
+  mrpDown: "mrpDown",
+  thumbUp: "thumbsUp",
+  point: "point",
+  fist: "allGrip",
+  pinch: "pinch"
+};
+
+// TODO: When we have analog values of index-finger triggers or middle-finger grips,
+//       it would be nice to animate the hands proportionally to those analog values.
+AFRAME.registerComponent("animated-robot-hands", {
+  schema: {
+    leftHand: { type: "selector", default: "#player-left-controller" },
+    rightHand: { type: "selector", default: "#player-right-controller" }
+  },
+
+  init: function() {
+    window.hands = this;
+    this.playAnimation = this.playAnimation.bind(this);
+
+    // Get the three.js object in the scene graph that has the animation data
+    const root = this.el.querySelector("a-gltf-entity .RootScene").object3D.children[0];
```

Comment:
We talked briefly about creating our own `animation-mixer` component that we can add to the root scene element and other components such as this one can reference. I'll need that for implementing the eye animations. I think this is fine for this PR, let's talk more about it later.